src/main/scala/system/ExampleRocketSystem.scala

```diff
@@ -16,12 +16,15 @@ class ExampleRocketSystem(implicit p: Parameters) extends RocketSubsystem
     with CanHaveMasterAXI4MemPort
     with CanHaveMasterAXI4MMIOPort
     with CanHaveSlaveAXI4Port
-    with HasPeripheryBootROM {
+{
+  // optionally add ROM devices
+  val bootROM  = p(BootROMLocated(location)).map { BootROM.attach(_, this, CBUS) }
+  val maskROMs = p(MaskROMLocated(location)).map { MaskROM.attach(_, this, CBUS) }
```

Comment:
@rmac-sifive These are candidate targets for a more generic `HasDevices` trait in rocket-chip, but note that the `BootROM` one needs more than just `Attachable`